webpack.config.js

```diff
@@ -7,10 +7,13 @@ const outputName = 'app-webpack';
 
 module.exports = ({ env = 'dev', plugins = [] } = {}) => ({
     devtool: env === 'dev' ? 'inline-source-map' : 'source-map',
-    entry: path.join(__dirname, 'static', 'src', 'javascripts', 'boot-webpack.js'),
+    entry: {
+        [outputName]: path.join(__dirname, 'static', 'src', 'javascripts', 'boot-webpack.js'),
+        admin: path.join(__dirname, 'static', 'src', 'javascripts-legacy', 'bootstraps', 'admin.js'),
+    },
     output: {
         path: path.join(__dirname, 'static', 'target', 'javascripts'),
-        filename: `${env === 'dev' ? '' : '[chunkhash]/'}${outputName}.js`,
+        filename: `${env === 'dev' ? '' : '[chunkhash]/'}[name].js`,
         chunkFilename: `${env === 'dev' ? '' : '[chunkhash]/'}${outputName}.chunk-[id].js`,
```

Comment:
no that's cool then. once require is gone we can look at doing them properly, naming the chunks. ill a thing to the project